src/WebJobs.Script/Diagnostics/FastLogger.cs

```diff
@@ -27,25 +27,13 @@ public FastLogger(string accountConnectionString)
             this._writer = LogFactory.NewWriter(containerName, table);
         }
 
-        // SDK notification gives us the full name, which came from Ref.Emit.
-        // It's 'Type.Method'.  We just want 'Method'
-        private static string GetShortName(string fullname)
```

Comment:
I had this in my local work branch already, so we'll get this in as well. This function was added recently by Mike, but we already have a Utility function that does the exact same thing.